sys/include/net/gnrc/netif/conf.h

```diff
@@ -182,8 +182,8 @@ extern "C" {
  *
  * @experimental
  *
- * This feature is non compliant with RFC 4944 and might not be supported by
- * other implementations.
+ * This feature is non compliant with RFC 4944, RFC 7668 and might not be
```

Comment:
Nitpick, but
```suggestion
 * This feature is non compliant with RFC 4944 and RFC 7668 and might not be
```

is grammatically more correct